skimage/filters/thresholding.py

```diff
@@ -970,6 +970,106 @@ def _mean_std(image, w):
     # m*m when floating point error is considered
     s = np.sqrt(np.clip(g2 - m * m, 0, None))
     return m, s
+ 
+def _only_mean(image, w):
```

Comment:
Rather than introduce a new function here, it would be better to modify the existing `_mean_std` function with a `mean_only` or `omit_std` argument that could be used to return the mean only. Then just return `m` (mean) early, before the computation of `s` (std).

Reply:
Ok, thanks for the suggestion.